Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.py

```diff
@@ -556,6 +575,65 @@ def get_external_ip_address_range_command(client: Client, args: dict[str, Any])
     return command_results
 
 
+def get_attack_surface_rule_command(client: Client, args: dict[str, Any]) -> CommandResults:
+    """
+    asm-list-external-service command: Returns list of external services.
+
+    Args:
+        client (Client): CortexXpanse client to use.
+        args (dict): all command arguments, usually passed from ``demisto.args()``.
+            ``args['ip_address']`` IP Address to search on.
+            ``args['domain']`` Domain to search on.
+            ``args['is_active']`` If the service active or not.
+            ``args['discovery_type']`` how service was discovered.
+
+    Returns:
+        CommandResults: A ``CommandResults`` object that is then passed to ``return_results``, that contains external
+        services.
+    """
+    attack_surface_rule_id = args.get("attack_surface_rule_id")
+    enabled_status = args.get("enabled_status")
+    priority = args.get("priority")
+    category = args.get("category")
+
+    search_params = []
+    if attack_surface_rule_id:
+        search_params.append({
+            "field": "attack_surface_rule_id",
+            "operator": "in",
+            "value": attack_surface_rule_id.split(",")
+        })
+    if enabled_status:
+        search_params.append({
+            "field": "enabled_status",
+            "operator": "in",
+            "value": enabled_status.split(",")
+        })
+    if priority:
+        search_params.append({
+            "field": "priority",
+            "operator": "in",
+            "value": priority.split(",")
+        })
+    if category:
+        search_params.append({
+            "field": "category",
+            "operator": "in",
+            "value": category.split(",")
+        })
+
+    response = client.get_attack_surface_rule_request(search_params)
+    parsed = response.get("reply", {}).get("attack_surface_rules")
+    command_results = CommandResults(
+        outputs_prefix='ASM.AttackSurfaceRule',
+        outputs_key_field='attack_surface_rule',
```

Comment:
it's look like `attack_surface_rule` didn't defined in the YML file and. in the results
do you mean `attack_surface_rule_id`